### DISCUSSION TITLE
Customizing installer?

### DISCUSSION THREAD

**Discussion by NikolaGrgic** [tauri-apps/tauri#2783] [Q&A] [answered]
Hi all,

How can we customize the installer that installs our built Tauri application? I'm looking to customize the background images, icons, etc. to resemble my brand instead of a generic solution. Also, on installation, I'd also like to create some default directories and settings files for use within the actual application later.

Any resources, guides, or ideas anyone could point me to?

**Comment by amrbashir** [comment, marked as answer]
I am guessing you mean the windows installer, so check this https://tauri.studio/en/docs/api/config#tauri.bundle.windows.wix there is also similar config for macOS and linux.

**Reply by jerrygreen** [reply]
It would be cool if Tauri had some example for custom installers, like those: https://github.com/tauri-apps/tauri/tree/dev/examples.

**Reply by wesin** [reply]
it seems so pool with so little examples. really pain to try and study wix. so pity
